code/game/objects/items/weapons/defib.dm

```diff
@@ -130,6 +130,8 @@
 /obj/item/weapon/melee/defibrillator/proc/attemptDefib(mob/living/carbon/human/target,mob/user)
 	user.visible_message("<span class='notice'>[user] starts setting up the paddles on [target]'s chest.</span>", \
 	"<span class='notice'>You start setting up the paddles on [target]'s chest</span>")
+	if(target.mind && !target.client && target.get_heart() && target.get_organ(LIMB_HEAD) && target.has_brain() && !target.mind.suiciding && target.health+target.getOxyLoss() > config.health_threshold_dead)
+		target.ghost_reenter_alert("Someone is about to try to defibrilate your body. Return to it if you want to be resurrected!")
```

Comment:
`target.ghost_reenter_alert("Someone is about to try to defibrillate your body. Return to it if you want to be resurrected!")`

Reply:
fixed typo